projects/frontend/shared-components/gui/version.txt

```diff
@@ -1 +1 @@
-0.0.1
+0.0
```

Comment:
use semantic version format, please find
https://github.com/vmware/versatile-data-kit/blob/main/CONTRIBUTING.md#changelog

Reply:
This file is supposed to represent the major and minor version. We take the patch version from the pipeline id. Omitting the patch version seemed like the simpler option. I can always make it say 0.0.0 and run `sed` on it to change it when we run.

Reply:
Format fixed, ended up going with `awk` instead.